<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>129</y>
      <width>881</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>731</width>
        <height>531</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>GestionCandidats</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_candidat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>datenaissance</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>numtel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_candidat">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>90</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_niveau">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>170</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>210</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Ajouter">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="lineEdit_datenaissance">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>130</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>niveau</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_numtel">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>250</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_CANDIDATS">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>280</y>
          <width>621</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Supprimer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Modifier">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_candidat</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_OffresEmplois">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>261</y>
        <width>631</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Supprimer2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>description</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ajouter2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID_OffreEmploi">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>titre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID_candidat_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>lieu</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="lineEdit_datepublication">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_OffreEmploi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>datepublication</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_titre">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>nbrplace</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_lieu">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_description">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID_nbrplace">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Modifier2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
